public/discussion.php

```diff
@@ -3,6 +3,7 @@
 require_once "../common.php";
 
 ensure_logged_in();
+auth_loggedin();
 
```

Comment:
Maybe the `auth_logged_in` method should be invoked in the `ensure_logged_in` function, what do you think

Reply:
I was actually conflicted about that, i see ensure logged in as a method make sure that the user is logged in if they want to see a certain page. But verify_logged_in is more of a cleanup method to make sure that theres no mismatch between the users cookies and the database in a way

Reply:
ok with that being the case, should `verify_logged_in` be in the header file then?

Reply:
this is a good idea